### ISSUE TITLE
'ExcelWorksheet.MergedCells' need improvement

### ISSUE THREAD

**Issue by kelicto** [EPPlusSoftware/EPPlus#660]
Problem 1: 
The original meaning expressed by the name is inconsistent with the actual meaning, not knowing the meaning by the name.

Problem 2: 
It is not always in order, which is fatal.

Code Reference:
``` csharp
public static List<ExcelRange> GetSortedMergedRanges(this ExcelWorksheet worksheet)
{
    if (worksheet is null)
        throw new ArgumentNullException(nameof(worksheet));

    return worksheet.GetSortedMergedRangeAddresses().Select(s => worksheet.Cells[s]).ToList();
}

public static List<string> GetSortedMergedRangeAddresses(this ExcelWorksheet worksheet)
{
    if (worksheet is null)
        throw new ArgumentNullException(nameof(worksheet));

    return worksheet.MergedCells.OrderBy(o => string.Join(string.Empty, o.Split(':').First().Reverse())).ToList();
}

public static double GetTotalWidth(this ExcelWorksheet worksheet)
{
    if (worksheet is null)
        throw new ArgumentNullException(nameof(worksheet));

    var result = 0d;

    for (var i = 0; i < worksheet.Dimension.Columns; i++)
        result += worksheet.Column(i + 1).Width;

    return result;
}
```

**Comment by JanKallman** [comment]
Problem 1:
Can you develop what the problem is?
Problem 2:
This collection is ordered as it's Loaded/Added. Why is this fatal?

**Comment by kelicto** [comment]
> Problem 1: Can you develop what the problem is?
The meaning is very vague and it is recommended to include "Addresses".

> Problem 2: This collection is ordered as it's Loaded/Added. Why is this fatal?

Because some cell Range is fixed content, I need to get it, so I have to make sure it is in order.
![image](https://user-images.githubusercontent.com/23723575/172078987-2c69cf84-8653-425f-ad45-8860d44679cd.png)

**Comment by JanKallman** [comment]
Yes, I can agree that addresses should be included in the name to make it more clear, but it will break code to change a public property name, so we will not do that. Sorting the addresses will also break code that access the index of this collection. Also note that Excel does not sort this collection when the workbook is saved. 
If you need them sorted it's easy enough to sort them as you do in your extension method above.

**Comment by kelicto** [comment]
Naming issues can be marked with obsolete features, and changes described in version updates, I think users can accept these, and many large companies do interface changes to solve problems. Of course you can not do, but please write the comments more detailed, necessary to have some examples, other properties have done examples, which is great.

I do not know what the sorting breaks, because from the objective facts of the order, rather than arbitrary sorting, if the internal code may do to adjust, I think that do it, so that the user does not have to do all this extra processing work, to reduce the burden of everyone. Of course, the worst thing you can do is to provide such extensions, so that other users don't have to write this code to do this desired work.
